ert_storage/endpoints/records.py

```diff
@@ -67,8 +77,13 @@ async def post_ensemble_record(
         .count()
         > 0
     ):
-        raise RuntimeError(
-            f"Ensemble-wide record '{name}' for ensemble '{ensemble_id}' already exists"
+        raise HTTPException(
+            status_code=404,
```

Comment:
Perhaps **409 Conflict** is a good status code here? https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/409
Full list of statuses: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status

Also, I think it's preferable if we use Starlette's status module:
```py
from fastapi import status

raise HTTPException(status_code=status.HTTP_409_CONFLICT, ...)
```

Reply:
Makes sense!